apps/src/templates/TeacherFeedback.jsx

```diff
@@ -22,12 +23,23 @@ const styles = {
   },
   content: {
     padding: 10
+  },
+  textInput: {
+    margin: 10,
+    display: 'block',
+    width: '100%',
+  },
+  button: {
+    margin: 10,
+    fontWeight: 'bold'
   }
 };
 
 class TeacherFeedback extends Component {
   static propTypes = {
-    viewAs: PropTypes.oneOf(['Teacher', 'Student'])
+    viewAs: PropTypes.oneOf(['Teacher', 'Student']),
+    //temp prop for which version to display (stable, released 2018-teacher-experience, or internal, developer version)
+    internalFlag: PropTypes.bool
```

Comment:
Having a comment here is great! Name could be a little more descriptive - maybe `withUnreleasedFeatures` or something?